sources/Core/StringUtils.h

```diff
@@ -131,8 +131,8 @@ LLGL_EXPORT void StringPrintf(std::string& str, const char* format, va_list args
 
 struct FormattedTableColumn
 {
-    unsigned                maxWidth        = -1;
-    unsigned                multiLineIndent = 0; // Number of blanks for each multi-line cell
+    std::uint32_t           maxWidth        = std::numeric_limits<std::uint32_t>::max();
```

Comment:
I am having mixed feelings about using `std::numeric_limits` outside a templated context and just using `UINT32_MAX` here. I am trying to reduce the amount of templated code in this project due to horrible compile times and readability (`UINT32_MAX` is concise and sufficient). On the other hand, it's intended to be a C++ project not relying on pure C definitions and I just realized `UINT32_MAX` is only optional - one of the many things that are loosely defined in the C++ standard -.-

I think this can stay `unsigned` as I don't see the necessity for a fixed size integer here, in which case you can change the `-1` to `UINT_MAX`, which is always defined in `<climits>`.